src/js/views/patients/sidebar/action-sidebar_views.js

```diff
@@ -1,10 +1,12 @@
 import _ from 'underscore';
 import Backbone from 'backbone';
 import hbs from 'handlebars-inline-precompile';
-import { View, CollectionView } from 'marionette';
+import { View } from 'marionette';
 
 import Optionlist from 'js/components/optionlist';
 
+import { ActivitiesView } from './action-sidebar_activity-views';
```

Comment:
is this right? Should this be imported here only to be exported? Feels not right.